UnityProject/Assets/Scripts/Systems/Fluids/MixAndVolume.cs

```diff
@@ -37,7 +37,7 @@ public float Temperature
 		{
 			get
 			{
-				if (WholeHeatCapacity == 0)
+				if (Mathf.Approximately(WholeHeatCapacity, 0))
```

Comment:
with these cases, it only needs to trigger if it is exactly 0, since there is a divide by coming up

Reply:
So this is to avoid dividing by zero? Might be a better way to handle that, maybe a try catch.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.dividebyzeroexception?view=net-5.0

Reply:
> Dividing a floating-point value by zero doesn't throw an exception; it results in positive infinity, negative infinity, or not a number (NaN), according to the rules of IEEE 754 arithmetic. 

shieeeeet

Reply:
alright, then maybe checking if the division result is NaN https://docs.microsoft.com/en-us/dotnet/api/system.single.isnan?redirectedfrom=MSDN&view=net-5.0#System_Single_IsNaN_System_Single_

Reply:
up to you guys MixAndVolume.cs is not really my code, I dont have counter arguments if you want to edit it but I dont want it to hold the pr

Reply:
humm,  would be nice and easy to fix if it's an issue so is saying it's okay